src/controllers/countries.mjs

```diff
@@ -0,0 +1,68 @@
+import { getFileData, writeFile } from "../utils/fileHelper.mjs";
+import { BASEDIR } from "../../config.mjs";
+
+export const getCountries = async (_, res) => {
+  try {
+    const countries = await getFileData(`${BASEDIR}/data/countries.json`);
+    res.status(200).json(countries);
+  } catch (error) {
+    res.status(500).json({ message: "Error fetching countries data" });
+  }
+};
+
+export const getCitiesInCountry = async (req, res) => {
+  try {
+    const cities = await getFileData(`${BASEDIR}/data/cities.json`);
+    const countryName = req.params.countryName.toLowerCase();
+    const countryCities = cities.filter(
+      (city) => city.country.toLowerCase() === countryName
+    );
+    if (countryCities.length > 0) {
+      res.json(countryCities);
+    } else {
+      res.status(404).json({ message: "No cities found for this country" });
```

Comment:
Maybe I would suggest to mention what exact country
`No cities found for ${countryName}`